cloudferrylib/os/image/glance_image.py

```diff
@@ -174,14 +174,18 @@ def get_tags(self):
     def get_members(self):
         # members structure {image_id: {tenant_name: can_share}}
         result = {}
-        for image in self.get_image_list():
-            for entry in self.glance_client.image_members.list(image=image.id):
-                if image.id not in result:
-                    result[image.id] = {}
+
+        for img in self.get_image_list():
```

Comment:
I think there is no real value in renaming `image` to `img`, please rollback this change.